config/none_ft/ccg.conf

```diff
@@ -0,0 +1,31 @@
+include "../defaults.conf"
+
+pretrain_tasks = ccg
+target_tasks= ccg
```

Comment:
Is there a reason this needs to be set in both? Can we just set it as a target task?

(Also, please be consistent with spacing.)